examples/batch.R

```diff
@@ -0,0 +1,121 @@
+# Batch examples
+
+# Note: In some cases, you'll need to wait for a step to complete in your AWS
+# account before you can successfully run the next step.
+
+# To set up the Batch compute environment, get security group and subnet info
+# for the default VPC.
+
+default_vpc <- paws.ec2::describe_vpcs(
+  Filters = "isDefault=true"
+)$Vpcs[[1]]
+
+security_group <- paws.ec2::describe_security_groups(
+  Filters = sprintf("vpc-id=%s", default_vpc$VpcId),
+  GroupNames = "default"
+)$SecurityGroups[[1]]
+
+subnets <- paws.ec2::describe_subnets(
+  Filters = sprintf("vpc-id=%s", default_vpc$VpcId)
+)$Subnets
+
+#-------------------------------------------------------------------------------
+
+# Set up an IAM role for Batch.
+
+role_name <- "TestBatchServiceRole"
+policy_arn <- "arn:aws:iam::aws:policy/service-role/AWSBatchServiceRole"
```

Comment:
@davidkretch the policy_arn still looks like this:
```
"arn:aws:iam::aws:policy/service-role/AWSBatchServiceRole"
```

However, the documentation says it should look like:

```
"arn:aws:iam::aws:policy/AWSBatchServiceRole"
```

Reply:
Where is the documentation you are referring to? When I look at the AWSBatchServiceRole policy, that is its ARN.

Reply:
If you look at the documentation we generate for the function `paws.batch::create_compute_environment`, it expects it in the format I am suggesting. I had to change it to the format I suggested to get it to work for me. Otherwise it said the compute environment was invalid, because it did not have a role attached

Reply:
Cool 👍 Let me know if this version works for you.